### PR TITLE
As a user I want to edit and delete my own stories

### PR DESCRIPTION
**Pull request by Dielone**

### CODE REVIEW

File: app/controllers/stories_controller.rb [hunk 1]
```diff
@@ -1,20 +1,46 @@
 class StoriesController < ApplicationController
-  expose(:story)
-
-  def index
-    @stories = Story.all
-  end
+  respond_to(:html)
+  expose_decorated(:story, attributes: :story_params)
+  expose(:stories)
```
Comment (vladimirbazhanov) [inline app/controllers/stories_controller.rb]: what this object will contain?

File: app/controllers/stories_controller.rb [hunk 2]
```diff
@@ -1,20 +1,46 @@
...
+    authorize story
+    if story.save
+      redirect_to(story)
     else
       render :new
     end
   end
 
+  def new
+    authorize story
```
Comment (vladimirbazhanov) [inline app/controllers/stories_controller.rb]: move to before_action ?

File: app/views/stories/edit.html.slim [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      h2 = title("Edit your story")
+
+  .row
+    .medium-12.columns
+      = simple_form_for story,
+          url: {action: "update"} do |f|
+
+        .form-inputs
+          = f.input :name, required: true, autofocus: true
+          = f.cktext_area :body, :cols => 40,:ckeditor => {:uiColor => '#67b1d5', :height => 300}, required: true
```
Comment (vladimirbazhanov) [inline app/views/stories/edit.html.slim]: pls do not copy and paste code from README)) use pls ruby 2.0 syntax without =>

**Comment by vladimirbazhanov** [comment]
👍

File: app/controllers/stories_controller.rb (hunk 1 above)
Comment (Dielone) [inline app/controllers/stories_controller.rb]: @vladimirbazhanov for now story will contain name and body from story_params.

File: app/views/stories/edit.html.slim (hunk 3 above)
Comment (Dielone) [inline app/views/stories/edit.html.slim]: @vladimirbazhanov oops, sorry)
I usually check code for these syntax disparities, but they appear often..magic

**Pull request merged**